platform/FEATURE_EXPERIMENTAL_API/FEATURE_PSA/TARGET_MBED_PSA_SRV/TESTS/attestation/test/CMakeLists.txt

```
@@ -1,18 +1,19 @@
 # Copyright (c) 2021 ARM Limited. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 
-cmake_minimum_required(VERSION 3.19.0 FATAL_ERROR)
+include(mbed_greentea)
 
-set(MBED_PATH ${CMAKE_CURRENT_SOURCE_DIR}/../../../../../../../ CACHE INTERNAL "")
-set(TEST_TARGET mbed-platform-psa-attestation)
-
-include(${MBED_PATH}/tools/cmake/mbed_greentea.cmake)
-
-project(${TEST_TARGET})
+if(MBED_GREENTEA_TEST_BAREMETAL)
+    set(TEST_SKIPPED "RTOS required")
+elseif(NOT ("PSA" IN_LIST MBED_TARGET_LABELS AND "EXPERIMENTAL_API" IN_LIST MBED_TARGET_LABELS))
+    set(TEST_SKIPPED "Experiemental API and PSA required")
```

```suggestion
    set(TEST_SKIPPED "Experimental API and PSA required")
```
Tiny typo on a line that sounds like it may not be needed, but just in case!

---

Good point, this line may not be needed. If a users builds test from the top of mbed-os, PSA tests are not even included unless PSA is enabled. This check is only needed when building from the test's own directory which is not recommended, so I'll remove this check.

---

Done